radiomics/__init__.py

```diff
@@ -32,6 +37,54 @@ def debug(debug_on=True):
     debugging = False
 
 
+def getFeatureClasses():
+  """
+  Iterates over all modules of the radiomics package using pkgutil and subsequently imports those modules.
+
+  Return a dictionary of all modules containing featureClasses, with modulename as key, abstract
+  class object of the featureClass as value. Assumes only one featureClass per module
+
+  This is achieved by inspect.getmembers. Modules are added if it contains a member that is a class,
+  with name starting with 'Radiomics' and is inherited from :py:class:`radiomics.base.RadiomicsFeaturesBase`.
+
+  This iteration only runs once (at initialization of toolbox), subsequent calls return the dictionary created by the
+  first call.
+  """
+  global _featureClasses
+  if _featureClasses is None:  # On first call, enumerate possible feature classes and import PyRadiomics modules
+    _featureClasses = {}
+    for _, mod, _ in pkgutil.iter_modules([os.path.dirname(base.__file__)]):
+      if str(mod).startswith('_'):  # Skip loading of 'private' classes, these don't contain feature classes
+        continue
+      __import__('radiomics.' + mod)
+      module = sys.modules['radiomics.' + mod]
+      attributes = inspect.getmembers(module, inspect.isclass)
+      for a in attributes:
+        if a[0].startswith('Radiomics'):
+          if base.RadiomicsFeaturesBase in inspect.getmro(a[1])[1:]:
+            _featureClasses[mod] = a[1]
+
+  return _featureClasses
+
+
+def getInputImages():
```

Comment:
To me this name is quite confusing. Wouldn't `getInputImageTypes` make more sense?